.github/CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # default PR reviews to the team
-* @hashicorp/team-consul-india
```

Comment:
@hashicorp/team-consul-india is deleted in [doormat](https://passport.hashicorp.services/namespaces/doormat/groups/team-consul-india)